Source/Library/Adapt.Models/Adapt.Models.csproj

```diff
@@ -17,11 +17,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Gemstone.Common" Version="1.0.67" />
-    <PackageReference Include="Gemstone.Data" Version="1.0.67" />
-  </ItemGroup>
-
-  <ItemGroup>
+    <ProjectReference Include="..\..\..\..\gemstone\common\src\Gemstone\Gemstone.Common.csproj" />
```

Comment:
Should be changed to package reference to ensure we have the most up to date package